docs/UserGuide.adoc

```diff
@@ -353,10 +360,13 @@ Alias: `fl`
 * Filter is case insensitive eg. `s1234567a` matches `S1234567A`
 ====
 Example:
-`fl S1213452A`
-====
-* List all leave application requested by the user with NRIC, 'S1213452A'.
-====
+`fl T1213452E`
```

Comment:
Btw for examples try to show the whole command instead of using Alias. Same for screenshots also.